Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.11-slim-buster
```

Comment:
Using this did slim down the image size quite significantly. But not as slim as an `alpine` image.

I tried setting up `alpine` image but ran into several issues trying to install the dependencies. Let's try setting up that image sometimes in the future for deployment. For now, this is good enough